luaui/Widgets/gui_pregameui.lua

```diff
@@ -79,10 +81,267 @@ local buttonRect = {0,0,0,0}
 local lockRect = {0,0,0,0}
 local blinkButton = false
 
+-- DraftOrder mod start
+local draftMode = Spring.GetModOptions().draft_mode
+local turnTimeOut = 5 -- This controls timeout for random/skill mode
+local VoteSkipTurnDelay = turnTimeOut+3
+local draftModeLoaded = false
+local DMDefaultColorString = '\255\200\200\200'
+local DMWarnColor = '\255\255\255\255'
+local myTurn = false
+local ihavejoined_fair = false
+local currentTurnTimeout = nil
+local voteSkipTurnTimeout = nil
+local fairTimeout = nil
+local current_playerID = -1
+local next_playerID = -1
+local auto_ready_disable = false
+local myTeamPlayersOrder = nil
+local currentPlayerIndex = 0
+local hasStartbox = false
+local devUItestMode = false -- flip to true to test UI with fake players
+-- a lot of code copied from advplayerlist...
+local imgDir = LUAUI_DIRNAME .. "Images/advplayerslist/"
+local imageDirectory = ":lc:" .. imgDir
+local pics = {
+    readyTexture = imageDirectory .. "indicator.dds",
+    rank0 = imageDirectory .. "ranks/1.png",
+    rank1 = imageDirectory .. "ranks/2.png",
+    rank2 = imageDirectory .. "ranks/3.png",
+    rank3 = imageDirectory .. "ranks/4.png",
+    rank4 = imageDirectory .. "ranks/5.png",
+    rank5 = imageDirectory .. "ranks/6.png",
+    rank6 = imageDirectory .. "ranks/7.png",
+    rank7 = imageDirectory .. "ranks/8.png",
+}
+local playerReadyState = {}
+local playerScale = 1
+local gl_Texture = gl.Texture
+local gl_Color = gl.Color
+local function RectQuad(px, py, sx, sy)
+    local o = 0.008        -- texture offset, because else grey line might show at the edges
+    gl.TexCoord(o, 1 - o)
+    gl.Vertex(px, py, 0)
+    gl.TexCoord(1 - o, 1 - o)
+    gl.Vertex(sx, py, 0)
+    gl.TexCoord(1 - o, o)
+    gl.Vertex(sx, sy, 0)
+    gl.TexCoord(o, o)
+    gl.Vertex(px, sy, 0)
+end
+local function DrawRect(px, py, sx, sy)
+    gl.BeginEnd(GL.QUADS, RectQuad, px, py, sx, sy)
+end
```

Comment:
This is very minor, but some whitespace for all these drawing functions will make it a bit more readable

Reply:
Added new lines.